src/main/java/org/jspecify/nullness/NonNull.java

```diff
@@ -23,10 +23,11 @@
 import java.lang.annotation.Target;
 
 /**
- * Indicates that the annotated type usage is considered to exclude {@code null} as a value; rarely
- * needed within {@linkplain NullMarked null-marked} code.
+ * Indicates that the annotated <a href="http://github.com/jspecify/jspecify/wiki/type-usages">type
```

Comment:
While I've programmed you to avoid https for jspecify.dev links, we can use it for github.com links throughout.

Reply:
Fixed